projects/commudle-admin/src/app/feature-modules/public-community/components/home-community/home-community.component.scss

```diff
@@ -22,34 +29,53 @@
 
     .details {
       h1 {
-        margin-top: 0;
+        font-weight: $fw-600;
+        margin: $space-28 0 $space-8 0;
+        line-height: 2.4rem;
+        font-size: $fs-32;
       }
 
       .membership {
         display: flex;
```

Comment:
Let's try to use some the common classes defined for flex utilities directly in the HTML